server/src/utils/mixerConnections/VMixMixerConnection.ts

```diff
@@ -215,9 +234,11 @@ export class VMixMixerConnection {
                             outputLevel > 0 &&
                             Math.abs(outputLevel - input.volume) > 0.01
                         ) {
-                            dispatch(
-                                storeFaderLevel(assignedFaderIndex, input.volume)
-                            )
+                            this.dispatch({

```

Comment:
This cannot be replaced outright, instead line 225 should be changed to be typed instead.

Reply:
My fault, will add types, and rename dispatch to dispatchAndSetUpdateState. 
Didn't notice:
```
const dispatch = (update: any) => {
                    store.dispatch(update)
                    sendUpdate = true
                }